homeassistant/components/flume/coordinator.py

```diff
@@ -0,0 +1,37 @@
+"""The IntelliFire integration."""
+from __future__ import annotations
+
+from pyflume import FlumeData
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import _LOGGER, DEVICE_SCAN_INTERVAL, DOMAIN
+
+
+class FlumeDeviceDataUpdateCoordinator(DataUpdateCoordinator[FlumeData]):
+    """Data update coordinator for an individual flume device."""
+
+    def __init__(self, hass: HomeAssistant, flume_device) -> None:
+        """Initialize the Coordinator."""
+        super().__init__(
+            hass,
+            name=DOMAIN,
+            logger=_LOGGER,
+            update_interval=DEVICE_SCAN_INTERVAL,
+        )
+
+        self.flume_device = flume_device
+
+    async def _async_update_data(self):
```

Comment:
```suggestion
    async def _async_update_data(self) -> None:
```

Reply:
AFAICT, we aren't actually using the data, its just updating the `flume_device` object